src/anyvar/utils/liftover_utils.py

```diff
@@ -150,14 +151,14 @@ def get_liftover_variant(variant_object: dict, anyvar: AnyVar) -> VrsVariation:
 
         - `AccessionConversionError`: If unable to lift over the variant's refget accession
     """
-    if not variant_object:
+    if not input_variant:
         raise MalformedInputError
 
-    refget_accession = get_nested_key(
-        variant_object, "location", "sequenceReference", "refgetAccession"
+    refget_accession = get_nested_attribute(
+        input_variant, "location", "sequenceReference", "refgetAccession"
     )
-    start_position = get_nested_key(variant_object, "location", "start")
-    end_position = get_nested_key(variant_object, "location", "end")
+    start_position = get_nested_attribute(input_variant, "location", "start")
+    end_position = get_nested_attribute(input_variant, "location", "end")
     if not refget_accession or not start_position or not end_position:
         raise UnsupportedVariantLocationTypeError
```

Comment:
Kind of wondering if we could just do:

```suggestion
    try:
        refget_accession = input_variant.location.sequenceReference.refgetAccession
        start_position = input_variant.location.start
        end_position = input_variant.location.end
    except AttributeError:
        raise UnsupportedVariantLocationTypeError
```

and then remove `get_nested_attribute`